homeassistant/components/system_bridge/media_player.py

```diff
@@ -0,0 +1,264 @@
+)
+
+from homeassistant.components.media_player import (
+    MediaPlayerDeviceClass,
+    MediaPlayerEntity,
+    MediaPlayerEntityDescription,
+    MediaPlayerEntityFeature,
+    MediaPlayerState,
+    RepeatMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SystemBridgeEntity
+from .const import DOMAIN
+from .coordinator import SystemBridgeCoordinatorData, SystemBridgeDataUpdateCoordinator
+
+STATUS_CHANGING: Final[str] = "CHANGING"
+STATUS_STOPPED: Final[str] = "STOPPED"
+STATUS_PLAYING: Final[str] = "PLAYING"
+STATUS_PAUSED: Final[str] = "PAUSED"
+
+REPEAT_NONE: Final[str] = "NONE"
+REPEAT_TRACK: Final[str] = "TRACK"
+REPEAT_LIST: Final[str] = "LIST"
+
+MEDIA_STATUS_MAP: Final[dict[str, MediaPlayerState]] = {
+    STATUS_CHANGING: MediaPlayerState.IDLE,
+    STATUS_STOPPED: MediaPlayerState.IDLE,
+    STATUS_PLAYING: MediaPlayerState.PLAYING,
+    STATUS_PAUSED: MediaPlayerState.PAUSED,
+}
+
+MEDIA_REPEAT_MAP: Final[dict[str, RepeatMode]] = {
+    REPEAT_NONE: RepeatMode.OFF,
+    REPEAT_TRACK: RepeatMode.ONE,
+    REPEAT_LIST: RepeatMode.ALL,
+}
+
+MEDIA_SET_REPEAT_MAP: Final[dict[RepeatMode, int]] = {
+    RepeatMode.OFF: 0,
+    RepeatMode.ONE: 1,
+    RepeatMode.ALL: 2,
+}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up System Bridge media players based on a config entry."""
+    coordinator: SystemBridgeDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    data: SystemBridgeCoordinatorData = coordinator.data
+
+    if data.media is not None:
+        async_add_entities(
+            [
+                SystemBridgeMediaPlayer(
+                    coordinator,
+                    MediaPlayerEntityDescription(
+                        key="media",
+                        translation_key="media",
+                        icon="mdi:volume-high",
+                        device_class=MediaPlayerDeviceClass.RECEIVER,
+                    ),
+                    entry.data[CONF_PORT],
+                )
+            ]
+        )
+
+
+class SystemBridgeMediaPlayer(SystemBridgeEntity, MediaPlayerEntity):
+    """Define a System Bridge media player."""
+
+    entity_description: MediaPlayerEntityDescription
+
+    def __init__(
+        self,
+        coordinator: SystemBridgeDataUpdateCoordinator,
+        description: MediaPlayerEntityDescription,
+        api_port: int,
+    ) -> None:
+        """Initialize."""
+        super().__init__(
+            coordinator,
+            api_port,
+            description.key,
+        )
+        self.entity_description = description
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self.coordinator.data.media is not None
```

Comment:
`available` is already defined by the parent coordinator entity. We should probably check that too via `super().available`.